FlowCrypt/Controllers/Attachment/AttachmentViewController.swift

```diff
@@ -59,21 +60,29 @@ final class AttachmentViewController: UIViewController {
         saveAndStartDownload(file: file)
     }
 
-    override func viewWillDisappear(_ animated: Bool) {
-        super.viewWillDisappear(animated)
-        remove(file: file)
-    }
-
     private func remove(file: FileType) {
-        Task {
-            try await filesManager.remove(file: file)
+        Task { [weak self] in
+            do {
+                try await filesManager.remove(file: file)
+                self?.navigationController?.popViewController(animated: true)
+            } catch {
+                showAlert(message: "could_not_remove_opened_attachment".localized) {
```

Comment:
Include the `error` message in the printed string to the user. Else when we inevitably get a screenshot from user or customer when this happened, we won't be able to debug it.

Just adding `+ "
(\(error.localizedDescription))"` or something similar should already help.

Reply:
I am not really sure that user needs the actual error from iOS, but it easy to add, I will.

Reply:
The user doesn't need it.

But the user will complain, take a screen shot and send it to me. If it's missing the iOS error, what can I possibly do with such report?

Reply:
We could use Crashlytics or similar to that to get the reports for crashes and these kind of errors.

Reply:
Yes, but when a particular user writes you with a screenshot, you have to match them up with the reports that may or may not be in Crashlytics. When it is much simpler to just surface it to the user.

Hiding errors from users is a bad habit too common in the apple ecosystem. When apple website itself tells me "something went wrong" when it should have said "your session expired, you should log in again" (it does that all the time) I feel like throwing my apple laptop through a window (of Apple offices). It's a bad engineering practice.